src/main/java/com/bunq/sdk/model/generated/MonetaryAccount.java

```diff
@@ -57,14 +58,19 @@
     return list(apiContext, userId, new HashMap<>());
   }
 
+  public static BunqResponse<List<MonetaryAccount>> list(ApiContext apiContext, Integer userId,
+      Map<String, String> params) {
+    return list(apiContext, userId, params, new HashMap<>());
+  }
+
   /**
    * Get a collection of all your MonetaryAccounts.
```

Comment:
Why sudanly `your` ? Please change this to `the user's..`

Reply:
generated